core/layouts/harris/harris.info

```diff
@@ -0,0 +1,19 @@
+name = Harris
+version = BACKDROP_VERSION
+backdrop = 1.x
+type = layout
+
+; Specify regions for this layout.
+regions[header]     = Header
+regions[top]        = Top
+regions[content]    = Content
+regions[sidebar]    = First Sidebar
+regions[sidebar2]   = Second Sidebar
```

Comment:
This name change of regions from "sidebar_second" to "sidebar2" will make moving between the legacy and new layouts more difficult, as blocks won't automatically be remapped. Although I see the value in renaming "sidebar_first" to "sidebar" for the same reason. I would probably solve the old->new layout switching problem by:

- Keeping "sidebar_first" and "sidebar_second" regions (the old region names).
- Add a check to `layout_settings_form_update_layout()` to check if there is a difference between "sidebar" and "sidebar_first", and set the mapping as needed.

Keeping with the existing layout region names might also make upgrading from Drupal 7 easier, or at least we won't need to modify our current upgrade tests so much.